setup.py

```diff
@@ -57,8 +58,19 @@ def finalize_options(self) -> None:
         mkpath(self.build_temp)
         copy_file(join(dirname(__file__), 'CMakeLists.txt'),
                   self.build_temp)
-        cmake = run(['cmake', '.'], check=True, stdout=DEVNULL, stderr=PIPE,
-                    cwd=self.build_temp, universal_newlines=True)
+        try:
+            cmake = run(['cmake', '.'],
+                        check=True, stdout=DEVNULL, stderr=PIPE,
+                        cwd=self.build_temp, universal_newlines=True)
+        except CalledProcessError as e:
+            print(f"\n\ncmake: Error occurred!\n\n"
+                  f"(Standard output)\n"
+                  f"{e.stdout}\n\n"
+                  f"(Standard error)\n"
+                  f"{e.stderr}\n",
+                  file=sys.stderr)
```

Comment:
The following seems to be sufficient:
```suggestion
            print(e.stderr, file=sys.stderr)
```

Reply:
@McSinyx I'm unfamiliar with cmake and if stdout could help to clarify what's going on if an error arise, so it's ok for me. 

Giving no clues about what the cmake error output is could create confusion in a distracted user, let me now if you are sure about this

Reply:
As shown in your screenshot, CMake actually prints `CMake Error at ...` when there's a problem and such output will always be delivered to stderr.  In addition the script to be executed here also prints to stderr, so there's never any output to stdout.